addresspoints/src/main/resources/application.conf

```diff
@@ -4,6 +4,38 @@ akka {
 	log-dead-letters-during-shutdown = on
 }
 
+kamon {
+  statsd {
+    hostname = "localhost"
```

Comment:
Why the double entry for `hostname` and `port`?  Is it like the first is a static default, and the second overrides it if present?